src/Moq/Protected/ProtectedMock.cs

```diff
@@ -35,16 +35,24 @@ public ProtectedMock(Mock<T> mock)
 
 		public ISetup<T> Setup(string methodName, params object[] args)
 		{
-			Guard.NotNullOrEmpty(methodName, nameof(methodName));
+			return Setup(methodName, null, false, args);
```

Comment:
Detail: Please change `Setup` &rarr; `this.Setup` here and in other places where you made a similar change, in order to stay consistent with (the majority of) the code base.